### ISSUE TITLE
Error running java with Spring Boot on GAE 

### ISSUE THREAD

**Issue by sw-dev2** [google-pay/passes-rest-samples#23]
Error:
java.lang.NoClassDefFoundError: Could not initialize class com.google.common.io.BaseEncoding

The application runs on a local Spring Boot deployment within the IDE IntelliJ IDEA. The error occurs only when deploying an exploded war artifact to Google App Engine. It also occurs on a Tomcat server.

Using com.google.api-client', name: 'google-api-client', version: '1.30.9'

**Comment by sw-dev2** [comment]
I noticed that the guava dependency resolves to the android version. I changed to the jre version and now the error is different:
Handler dispatch failed; nested exception is java.lang.NoSuchMethodError: com.google.common.base.Preconditions.checkArgument(ZLjava/lang/String;C)V"}}

Looks like it is the same issue as SeleniumHQ/selenium#3880. Either it is reappearing or it has not been solved as mentioned there.

Another unsolved similar issue: https://groups.google.com/forum/#!topic/selenium-users/U7bOqbZ7vAk
